experiment/nursery/create_bazel_cache_rcs.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+# Copyright 2018 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# THIS IS TEMPORARY (!)
+# Once this moves out of experiment/ we will create these files in the bazel
+# images instead (!)
+# TODO(bentheelder): verify that this works and move it into the images
+CACHE_HOST="http://bazel-cache:8080"
+
+make_bazel_rc () {
+    echo "startup --host_jvm_args=-Dbazel.DigestFunction=sha256" >> $1
+    echo "build --spawn_strategy=remote" >> $1
+    echo "build --strategy=Javac=remote" >> $1
+    echo "build --genrule_strategy=remote" >> $1
+    echo "build --remote_rest_cache=${CACHE_HOST}" >> $1
+}
+
+# https://docs.bazel.build/versions/master/user-manual.html#bazelrc
+# bazel will look for two RC files, taking the first option in each set of paths
+# firstly:
+# - The path specified by the --bazelrc=file startup option. If specified, this option must appear before the command name (e.g. build)
+# - A file named .bazelrc in your base workspace directory
+# - A file named .bazelrc in your home directory
+make_bazel_rc "${HOME}/.bazelrc"
```

Comment:
I think this one isn't going to be read because we have a `.bazelrc` in the root of our workspace. (Which is something we need to fix...)

Reply:
The second one should be though, since we don't set `tools/bazel.rc`

This one is more for completeness. Bazel seems to have two bazelrc search paths from which it will take the first file, I'm creating our config at each of the last / "host machine level" locations.

This or something similar should be reusable for other repos in the future if things go well, some repos seem to use the `tools/bazel.rc` option.